### PR TITLE
 docker compose file

### PR DESCRIPTION
**Pull request by zemlyanoi**
temporary docker repository (mzemlyanoi/case-management-api) has to be changed to cwds/case-management-api

### CODE REVIEW

**Review by khilkevych — APPROVED** [review]
Great!

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+version: '3.1'
+services:
+  db2-ibm:
+      image: cwds/db2
+      ports:
+          - "50000:50000"
+      environment:
+          DB2INST1_PASSWORD: "db2inst1"
```
Comment (igor-chornobay-engagepoint) [inline docker/docker-compose.yml]: password should be db2inst1-pwd

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+          LICENSE: "accept"
+          DB2NAME: "cmsdb"
```
Comment (igor-chornobay-engagepoint) [inline docker/docker-compose.yml]: DB2NAME should be DB0TDEV. this should be a default.

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      command: db2start
+
+  case-api:
+      image: mzemlyanoi/case-management-api
```
Comment (igor-chornobay-engagepoint) [inline docker/docker-compose.yml]: Should use variable as a repository ref.

File: docker/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      hostname: case-management-api
+      depends_on:
+        - db2-ibm
+      ports:
+        - "8080:8080"
+        - "9081:9081"
+      environment:
+         SHIRO_CONFIG_PATH: "/opt/case-management-api/config/shiro.ini"
```
Comment (igor-chornobay-engagepoint) [inline docker/docker-compose.yml:22]: I think we should put here all the env variables, but comment them. This is a way to overwrite yml config

**Review by igor-chornobay-engagepoint — APPROVED** [review]

**Pull request merged**